index.html

```diff
@@ -189,22 +189,26 @@ <h3>The DDD "onion"</h3>
               width="500"
               alt="DDD Onion - Jeffrey Palermo"
             />
+              <aside class="notes">
+                  <ul>
+                      <li>Al centro il domain model, qui risiede la logica sottoforma di aggregati</li>
+                      <li>I servizi dell'applicazione, conoscono e interagiscono con gli aggregati per farli evolvere</li>
+                      <li>All'esterno abbiamo la UI/interfaccia di interazione con l'applicazione, i test, che mimano l'utente non conoscendo nulla dell'applicazione</li>
+                      <li>e infine l'infrastruttura, che è solo uno strumento per fare cose, che il db sia un file di testo o un server relazionale la logica di dominio non deve essere influenzata</li>
+                  </ul>
+              </aside>
           </section>
           <section>
             <h3>Presentation: API</h3>
             <h5>Open API + Stoplight</h5>
             <img
               style="float: left"
-              width="500"
-              src="img/stoplight.png"
+              width="700"
```

Comment:
This doesn't seem a flexible way to size the images. You should probably use inline CSS and tell both the images to use a ~47% width